xml/acls-2.1.rng

```diff
@@ -22,6 +23,7 @@
           <element name="acl_group">
             <!-- Here 'id' is the name of a unix group -->
             <attribute name="id"><data type="ID"/></attribute>
+            <attribute name="name"><text/></attribute>
```

Comment:
The new attributes should be optional. The tests should succeed after fixing that

Reply:
Hmm, still 70 failures.